pkg/ui/ccl/src/views/reports/containers/stores/encryption.tsx

```diff
@@ -1,10 +1,7 @@
 import React from "react";
 
 import * as protos from "src/js/protos";
-
-interface EncryptionStatusProps {
-  store: protos.cockroach.server.serverpb.StoreDetails$Properties;
-}
+import EncryptionStatusProps from "oss/src/views/reports/containers/stores/encryption";
```

Comment:
I guess I don't understand how ccl/oss import works. I thought CCL took precedence and the OSS stuff wouldn't even exist.

Reply:
Yeah, it's kind of confusing. "src/FOO" means "give me the CCL version, if it exists, otherwise the OSS version." You can specifically request the CCL or the OSS version by prefixing with "ccl/" or "oss/", respectively.

Reply:
My theory is that there's never a good reason to explicitly prefix `ccl`, and you should only ever prefix `oss` inside a file of the same name in the CCL code.